app/models/agreement_letter.rb

```diff
@@ -103,4 +103,11 @@ def unable_to_open? file
       true
     end
   end
+
+  def self.get_attachment
```

Comment:
I would leave this as get agreement letter attachment, since it is more precise and other attachments in emails are very likely